build/update-attribution-files/make_attribution.sh

```diff
@@ -44,7 +44,7 @@ function build::attribution::generate(){
 
 # Some projects have specific folders for different kubernetes release
 # dynamically find all versions to avoid having to update with every release
-RELEASE_FOLDER=$(find $PROJECT_ROOT -type d -name "1-*")
+RELEASE_FOLDER=$(find $PROJECT_ROOT -type d -maxdepth 1 -name "1-*")
```

Comment:
kind attributions werent being update because it was finding the 1-21 folder in the _output dir.